### ISSUE TITLE
Uncaught TypeError: Parser is not a constructor

### ISSUE THREAD

**Issue by mikiboy** [skyhookadventure/appsync-template-tester#10]
I've followed your usage example, but I am getting this error..

I've enabled `type: module` in root `package.json` and Here is script of root `package.json`:
```
"scripts": {
    "test": "NODE_OPTIONS=--experimental-vm-modules npx jest"
  }
```

Here is the test I am using:
```
import Parser from "appsync-template-tester";
import { readFileSync } from "fs";
import path, { join } from "path";
import { fileURLToPath } from "url";

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);

// Load a VTL template file to test
const templateFilePath = join(__dirname, "../../services/appsync/mapping-templates/Query.me.request.vtl");
const template = readFileSync(templateFilePath);

// Create the resolver
const parser = new Parser(template); // Note: this line throws this exception

test("Query.me.request.vtl", () => {
  // The Context (ctx) object for resolver
  const context = {
    result: {
      id: "testId",
    },
  };

  // parser.resolve() automatically typecasts (note JSON becomes a JS object for ease of testing)
  const response = parser.resolve(context);

  // For convenience, the response is returned as a JS object rather than JSON
  expect(response.id).toBe("testId");
});

```

**Comment by mikiboy** [comment]
I could fix this by trying like below:
```
const parser = new Parser.default(template);
```

Instead of `const parser = new Parser(template);`

**Comment by alan-cooney** [comment]
Fixed as per #11
